processing_handler.go

```diff
@@ -370,31 +371,28 @@ func handleProcessing(reqID string, rw http.ResponseWriter, r *http.Request) {
 	checkErr(ctx, "timeout", router.CheckTimeout(ctx))
 
 	if originData.Type == po.Format || po.Format == imagetype.Unknown {
-		// Don't process SVG
-		if originData.Type == imagetype.SVG {
-			if config.SanitizeSvg {
-				sanitized, svgErr := svg.Sanitize(originData)
-				checkErr(ctx, "svg_processing", svgErr)
-
-				// Since we'll replace origin data, it's better to close it to return
-				// it's buffer to the pool
-				originData.Close()
-
-				originData = sanitized
+		shouldSkipProcessing := slices.Contains(po.SkipProcessingFormats, originData.Type)
+
+		// Skip processing svg with unknown or the same destination imageType
+		// if it's not forced by AlwaysRasterizeSvg option
+		// Also skip processing if the format is in SkipProcessingFormats
+		if originData.Type == imagetype.SVG && !config.AlwaysRasterizeSvg || shouldSkipProcessing {
+			if originData.Type == imagetype.SVG {
+				if config.SanitizeSvg {
+					sanitized, svgErr := svg.Sanitize(originData)
+					checkErr(ctx, "svg_processing", svgErr)
+
+					// Since we'll replace origin data, it's better to close it to return
+					// it's buffer to the pool
+					originData.Close()
+
+					originData = sanitized
+				}
 			}
 
 			respondWithImage(reqID, r, rw, statusCode, originData, po, imageURL, originData)
 			return
 		}
```

Comment:
If we're simplifying things here, we could've simplify 'em even more 🙂

```go
shouldSkipProcessing := (originData.Type == po.Format || po.Format == imagetype.Unknown) &&
	(slices.Contains(po.SkipProcessingFormats, originData.Type) ||
		originData.Type == imagetype.SVG && config.AlwaysRasterizeSvg)

if shouldSkipProcessing {
	if config.SanitizeSvg {
		// ... do SVG stuff
	}

	respondWithImage(reqID, r, rw, statusCode, originData, po, imageURL, originData)
	return
}
```

Reply:
done
Only thing that I changed is 
```
config.AlwaysRasterizeSvg --> !config.AlwaysRasterizeSvg
```